### PR TITLE
fix remove no-schema config

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Fix some inconsistent expressions introduced in https://github.com/pingcap/docs-cn/pull/8144.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: get-started-with-tidb-lightning.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ TiDB Lightning 是一个将全量数据高速导入到 TiDB 集群的工具，
...
 
-在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.0.0 版本为例，具体部署方法可参考[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
+在开始数据导入之前，需先部署一套要进行导入的 TiDB 集群。本教程以 TiDB v5.4.0 版本为例，具体部署方法可参考[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
 
 ### 第 2 步：下载 TiDB Lightning 安装包
 
 通过以下链接获取 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）：
 
-- **v5.3.0**: [tidb-toolkit-v5.3.0-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.3.0-linux-amd64.tar.gz)
+- **v5.4.0**: [tidb-toolkit-v5.4.0-linux-amd64.tar.gz](https://download.pingcap.org/tidb-toolkit-v5.4.0-linux-amd64.tar.gz)
```
Comment (ran-huang) [inline get-started-with-tidb-lightning.md:50]: @sunzhaoyang Please confirm whether to use v6.0.0 in the example.

Comment (sunzhaoyang) [reply get-started-with-tidb-lightning.md:50]: No. I will replace this part with tiup later.

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**